src/foam/mlang/expr/TimeOfDay.js

```diff
@@ -45,6 +45,32 @@ foam.CLASS({
   methods: [
     {
       name: 'f',
+      code: function() {
+        var typeToIdx = {
+          year: 0,
+          month: 1,
+          day: 2
+        };
+        var zonetime = new Intl.DateTimeFormat(undefined, {
```

Comment:
Can we use 'default' instead of `undefined`? 
```suggestion
        var zonetime = new Intl.DateTimeFormat('default', {
```
@jlhughes, the default locale would suffice as the locale here is for formatting the data and time for the parser below (lines 61-69) so that we can extract the date parts appropriately.